### ISSUE TITLE
why encoder parts output size 2048 became 128 before enter the forward feedback NN

### ISSUE THREAD

**Issue by fpfcap** [cuhk-haosun/doc-journal-club#11]
why

**Comment by fpfcap** [comment]
i find that it's actually a dimension decrease which is not mentioned in the article. it can be done by using the neural classifier ,which is a part of the forward feedback neural network. Due to the set of nodes number, the classifier select the 2048 features and remain 128 of them. Since the project code is not opened so that i cannot find too precise detail to explain how the neural classifier work as well as the selection principle in this project. Commonly, the network select the features based on the loss function or baseline gradient, etc.

**Comment by fpfcap** [comment]
@xxx58

**Comment by xxx58** [comment]
got it. thx
